exercises/practice/bob/test.pony

```diff
@@ -16,9 +16,13 @@ class iso _TestBobShout is UnitTest
   fun apply(h: TestHelper) =>
     let shout_resp: String = "Whoa, chill out!"
 
+    // Inputs we expect to respond with "Whoa, chill out!"
     h.assert_eq[String](shout_resp, Bob("WATCH OUT!"))
+    // Inputs we expect to respond with "Whoa, chill out!"
     h.assert_eq[String](shout_resp, Bob("WHAT THE HELL WERE YOU THINKING?"))
+    // Inputs we expect to respond with "Whoa, chill out!"
     h.assert_eq[String](shout_resp, Bob("1, 2, 3 GO!"))
+    // Inputs we expect to respond with "Whoa, chill out!"
```

Comment:
A single comment, since the comment does say the plural "inputs" rather than the singular, would suffice above these assertions.  But the variable set matches the expectation, the code seems to be clear enough on its own without the comments.

Reply:
The same with other tests having this same pattern.